python.d/redis.chart.py

```diff
@@ -18,10 +18,15 @@
 #             'unix_socket': None
 #          }}
 
-ORDER = ['operations', 'hit_rate', 'memory', 'keys', 'net', 'connections', 'clients', 'slaves', 'persistence',
+ORDER = ['operations', 'hit_rate', 'memory', 'keys', 'net', 'connections', 'clients', 'eviction', 'slaves', 'persistence',
          'bgsave_now', 'bgsave_health']
 
 CHARTS = {
+    'eviction':{
+        'options': [None, 'Evicted keys', 'keys/s', 'evicted', 'redis.evictios', 'line'],
```

Comment:
`Evicted Keys`
Are you sure about `absolute`? If so then `keys/s` is wrong should be `keys`.